client/app/visualizations/chart/plotly/utils.js

```diff
@@ -248,7 +253,7 @@ function preparePieData(seriesList, options) {
     }, 0);
     each(serie.data, (row) => {
       const x = normalizeValue(row.x);
-      const y = normalizeValue(row.y);
+      const y = cleanNumber(normalizeValue(row.y));
```

Comment:
This is not used in case of bubble charts?

Reply:
It's only for pie charts (y there can be only numeric). Also I noticed a bug nearby - so fixed and updated PR